Frameworks/CoreText/CTRun.mm

```diff
@@ -117,7 +116,13 @@ void CTRunGetPositions(CTRunRef run, CFRange runRange, CGPoint* outPositions) {
         runRange.length = curRun->_range.length;
     }
 
-    memcpy(outPositions, &curRun->_glyphOrigins[runRange.location], sizeof(CGPoint) * runRange.length);
+    if (runRange.location != 0) {
+        UNIMPLEMENTED();
+        return;
+    }
+
+    outPositions->x = curRun->_xPos;
```

Comment:
could you open a github issue for this so this doesnt fall through the cracks? 

---
In reply to: [77871237](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77871237) [](ancestors = 77871237,77757825)